Mailer/app/api/v1/Handlers/Mailers/MailLogsCountHandler.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Remp\MailerModule\Api\v1\Handlers\Mailers;
+
+use ParseLogFilterConditionsTrait;
+use Remp\MailerModule\Api\Validation\InputValidatorTrait;
+use Remp\MailerModule\Repository\LogsRepository;
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+
+class MailLogsCountHandler extends BaseHandler
+{
+    private $logsRepository;
+
+    use InputValidatorTrait;
+    use ParseLogFilterConditionsTrait;
+
+    public function __construct(
+        LogsRepository $logsRepository
+    ) {
+        parent::__construct();
+        $this->logsRepository = $logsRepository;
+    }
+
+    public function handle($payload)
+    {
+        $request = file_get_contents("php://input");
+        $payload = $this->validateInput($request, 'logs-count.schema.json');
```

Comment:
* Please put schemas to the same folders as handlers. It's to logical place to search them for.
* `InputValidationTrait` should be dummy enough to only get JSON, path to schema (full path) and do the validation. It shouldn't be doing magic like searching for schemas in `schemas/` subfolder.